src/main/java/com/openlattice/edm/PostgresEdmManager.java

```diff
@@ -226,6 +224,37 @@ public void revoke(
         }
     }
 
+    public Iterable<UUID> getAllEntitySetIdsForType( UUID entityTypeId ) {
+        final var getEntitySetIdsByType = String.format( "SELECT %1$s FROM %2$s WHERE %3$s = ?",
+                ID.getName(),
+                ENTITY_SETS,
+                ENTITY_TYPE_ID_FIELD );
+        return new PostgresIterable<>(
```

Comment:
Is there a reason this is read from postgres instead of hazelcast?

Reply:
Now, that you mention it, I don't see any reason, just wanted to have a version, where we don't return the whole entity set, only the id. I will turn it into a hazelcast map query with filtering on entity type index. 
Btw, we have a lot of calls from EdmService to PostgresEdmManager.getAllEntitySetsForType, although there is a version in EdmService, which does exactly the same but using hazelcast. I will change those calls too to use hazelcast.

Reply:
Yeah avoiding loading the entire entity set is for sure ideal